android/app/src/main/java/com/contacttexter/CommunicationsModule.java

```diff
@@ -36,9 +36,8 @@ public void createPhoneCall(String number) {
 
     @ReactMethod
     public void createSMSMessage(String number, String message) {
-        Intent smsIntent = new Intent(Intent.ACTION_VIEW);
-        smsIntent.setType("vnd.android-dir/mms-sms");
-        smsIntent.putExtra("address", number);
+        Uri uri = Uri.parse("smsto:" + number);
```

Comment:
This did it huh?

What version are you running? Wonder if it's different between the different API versions.